src/host/services/IMUService.h

```diff
@@ -21,20 +21,49 @@
   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
   THE SOFTWARE.
 */
-
+#pragma once
 #include <Adafruit_BNO055.h>
 #include "common/os/Task.h"
 #include "common/signalk/SKHub.h"
+#include "host/config/IMUConfig.h"
 
 class IMUService : public Task {
   private:
+    IMUConfig &_config;
     SKHub &_skHub;
     Adafruit_BNO055 bno055;
-    uint8_t sysCalib, gyroCalib, accelCalib, magCalib;
+    uint8_t _sysCalib, _gyroCalib, _accelCalib, _magCalib;
+    uint8_t _axisConfig, _signConfig;
```

Comment:
Why did you made these two member variables of the IMUService class? I do not think we need them to be here. They can be local variables when we initialize the sensor like they were before.

Reply:
Because I was not aware, that it makes a lot of difference.
Sure I will bring them back!

Reply:
It just makes the object a little bigger in memory. These classes are allocated on the heap when we start the program. The local variables inside functions are allocated on the stack. If possible, it's better to use stack than heap for us.